compiler/rustc_attr_parsing/src/context.rs

```diff
@@ -502,10 +502,11 @@ impl<'f, 'sess: 'f, S: Stage> AcceptContext<'f, 'sess, S> {
         })
     }
 
+    /// produces an error along the linds of `expected one of [foo, meow]`
```

Comment:
meow

```suggestion
    /// produces an error along the lines of `expected one of [foo, meow]`
```

(And below)